### ISSUE TITLE
Generate ical (or equivalent) from course schedule for term

### ISSUE THREAD

**Issue by jasonhemann** [ryanhugh/searchneu#61]
It would be really useful to have an iCal download per course, or for faculty members, all of a user's courses for a term.

**Comment by ryanhugh** [comment]
I agree that this would be a good feature and I would totally use it myself. We could add another icon to the classes panel that exported the class to iCal or Google Calendar.

Courseoff has a similar feature and we could copy some ideas from how their feature works. If you want to work on this lmk! I'd be happy to point you in the right direction.

**Comment by edward-shen** [comment]
So uh the new banner actually supports exporting to iCal: 
![image](https://user-images.githubusercontent.com/6173958/45405207-f1b9bd80-b62f-11e8-8b3d-bfb4f288289b.png)
It's a little annoying, but it sends an iCal file to your husky email.

**Comment by jasonhemann** [comment]
@edward-shen, I didn't know about that feature (I think I'm not the only one)! Great find! Happy to close this out as already solved.

**Comment by jasonhemann** [comment]
Hm... well, looking at it, I don't know that faculty have that view.
